### PR TITLE
Feature/add test mode

### PR DESCRIPTION
**Pull request by aleerizw**

### CODE REVIEW

File: ssm.go [hunk 1]
```diff
@@ -12,12 +12,12 @@ import (
 // Checks if only 1 argument is provided
 // Returns the first command line argument
 func parseInput(args []string) (string, error) {
```
Comment (ahelal) [inline ssm.go]: if possible use flags

File: ssm.go [hunk 2]
```diff
@@ -39,17 +39,40 @@ func getParamValue(paramOutput *ssm.GetParameterOutput) string {
...
 	if err != nil {
 		log.Fatal(err)
 	}
-
-	out, err := retriveParam(paramName)
-	if err != nil {
-		log.Println("There was an error fetching/decrypting the parameter:", paramName)
-		log.Fatal(err.Error())
+	// Not test mode
+	if !*testMode {
```
Comment (ahelal) [inline ssm.go]: invert the logic :)

File: ssm.go (hunk 1 above)
Comment (aleerizw) [inline ssm.go]: Args() returns a slice of strings, so I am not sure how I can change the function
https://golang.org/pkg/flag/#Args

File: ssm.go (hunk 2 above)
Comment (aleerizw) [inline ssm.go]: Done

**Comment by ahelal** [comment]
+1

**Pull request merged**